tests/test_event.py

```diff
@@ -11,6 +40,17 @@ def test_eventedit():
     assert b"Edit Event" in response.content
 
 
+def test_eventedit_post_correct(user):
+    response = client.post("/event/edit", data=CORRECT_EVENT_FORM_DATA)
+    assert response.status_code == HTTP_303_SEE_OTHER
+    assert '/event/view/' in response.headers['location']
+
+
+def test_eventedit_post_wrong(user):
+    response = client.post("/event/edit", data=WRONG_EVENT_FORM_DATA)
+    assert response.json()['detail'] == 'VC type with no valid zoom link'
+
+
 def test_eventview_with_id():
     response = client.get("/event/view/1")
     assert response.status_code == 200
```

Comment:
Don't forget to use  starlette.status :)

Reply:
I didn't, whoever wrote it did:)

Reply:
So it could be great if you improve the existing code :)